umbraco/Umbraco-CMS
# v8: Updated search component to get translated placeholder "Search.." and "Clear" text

### Prerequisites
Link to issue: https://github.com/umbraco/Umbraco-CMS/issues/7121

### Description
- Updated search component to get translated placeholder "Search.." and "Clear" text.

### Steps to test:
1. Login to backoffice and select user language "Swedish".
2. Reload page for all Umbraco translations to kick in.
3. Click the search icon in top right corner. Placeholder should now say "Skriv för att söka..." and Clear text should now say "Rensa".

I added en.xml, en_us.xml and sv.xml translation for @general_clear, since these are the languages I know. Other languages will need to be updated with Clear translation over time, but since it fallback to english and the component is hardcoded english today, this should not be blocked by missing translations.

I've included some sceenshots of the end result. The logged in user is Swedish.
![after_search](https://user-images.githubusercontent.com/6122154/68568926-25bf8700-045d-11ea-964e-48a79285bd2c.png)
![after_clear](https://user-images.githubusercontent.com/6122154/68568920-235d2d00-045d-11ea-9110-7dffb8ad7578.png)

Cheers.
